src/modules/components/address/addressInput/addressInput.tsx

```diff
@@ -18,6 +18,7 @@ import {
 import type { IWeb3ComponentProps } from '../../../types';
 import { addressUtils, ensUtils } from '../../../utils';
 import { MemberAvatar } from '../../member';
+import { useOdsModulesContext } from '../../odsModulesProvider/odsModulesProvider';
```

Comment:
All `odsModulesProvider` imports can/should be changed to:
```suggestion
import { useOdsModulesContext } from '../../odsModulesProvider';
```

Reply:
✅ https://github.com/aragon/ods/pull/221/commits/4e26e3dd159f2792c0c16d5ff696d111f9c25635